crates/sui-core/src/authority.rs

```diff
@@ -1142,13 +1142,12 @@ impl AuthorityState {
             transaction_digest,
             protocol_config,
         );
-        let mut gas_status = SuiGasStatus::new_with_budget(
+        let gas_status = SuiGasStatus::new_with_budget(
             max_tx_gas,
             GasPrice::from(gas_price),
             storage_gas_price.into(),
             SuiCostTable::new(protocol_config),
         );
-        gas_status.charge_min_tx_gas()?;
```

Comment:
we are removing min charge because with buckets we do not need it anymore.
It also makes the code a bit more clear in terms of where charges happen